APISpecification.yaml

```diff
@@ -34,6 +34,62 @@ paths:
                 properties:
                   genome_stats:
                     $ref: '#/components/schemas/SpeciesStats'
+  /api/metadata/popular_species:
+    get:
+      description: List of 42 groups of genomes related to popular species
+      parameters:
+        - in: query
+          name: selected_genome_ids
+          schema:
+            type: string
+            example: 'a7335667-93e7-11ec-a39d-005056b38ce3,704ceb1-948d-11ec-a39d-005056b38ce3'
```

Comment:
I don't know how best to encode an array of strings, or whether sending an array of genome ids with a query suggests that a `POST` request will be more appropriate than a `GET` one.

For genome search api, we repeated a parameter name (`exclude`, see [source](https://github.com/Ensembl/ensembl-2020-genome-search/blob/522fd96470a2f2373d11950b4f62f201fd0ac61b/blueprints/genome_search.py#L34)). Example url: `https://staging-2020.ensembl.org/api/genomesearch/genome_search?query=hum&limit=20&exclude=a7335667-93e7-11ec-a39d-005056b38ce3&exclude=3704ceb1-948d-11ec-a39d-005056b38ce3&exclude=a73356e1-93e7-11ec-a39d-005056b38ce3&exclude=a73351f7-93e7-11ec-a39d-005056b38ce3&exclude=a73357ab-93e7-11ec-a39d-005056b38ce3&exclude=a733550b-93e7-11ec-a39d-005056b38ce3`.

Reply:
Why do we need to send this list of genome_ids in the first place ?

Reply:
See my reply to your other comment.